includes/functions-bindings-event-hubs.md

```diff
@@ -386,9 +386,8 @@ The following table explains the binding configuration properties that you set i
 |**path** |**EventHubName** | Functions 1.x only. The name of the event hub. When the event hub name is also present in the connection string, that value overrides this property at runtime. |
 |**eventHubName** |**EventHubName** | Functions 2.x and higher. The name of the event hub. When the event hub name is also present in the connection string, that value overrides this property at runtime. |
 |**consumerGroup** |**ConsumerGroup** | An optional property that sets the [consumer group](../articles/event-hubs/event-hubs-features.md#event-consumers) used to subscribe to events in the hub. If omitted, the `$Default` consumer group is used. |
-|**cardinality** | n/a | For Javascript. Set to `many` in order to enable batching.  If omitted or set to `one`, single message passed to function. |
+|**cardinality** | n/a | For Javascript. Set to `many` in order to enable batching.  If omitted or set to `one`, a single message is passed to the function. |
 |**connection** |**Connection** | The name of an app setting that contains the connection string to the event hub's namespace. Copy this connection string by clicking the **Connection Information** button for the [namespace](../articles/event-hubs/event-hubs-create.md#create-an-event-hubs-namespace), not the event hub itself. This connection string must have at least read permissions to activate the trigger.|
-|**path**|**EventHubName**|The name of the event hub. Can be referenced via app settings `%eventHubName%`|
```

Comment:
Duplicate. Both **path** and **EventHubName** are described above in this same table.

Reply:
Can you please add the statement:

>  Can be referenced via app settings `%eventHubName%`

to the remaining listing for `path`? I don't want to lose that content while removing the duplicate 😉